data/json/items/ammo/22.json

```diff
@@ -55,7 +57,7 @@
     "type": "AMMO",
     "name": ".22 rat-shot",
     "name_plural": ".22 rat-shot",
-    "description": "Rat-shot is an extremely weak .22 shotgun cartridge, designed for killing rats, snakes, or other small vermin while being unable to damage walls.  It has an extremely short range and is unable to injure all but the smallest creatures.",
+    "description": "A .22 caliber cartridge loaded with very small pieces of shot contained within a fragmenting plastic capsule.  It has an extremely short range and penetration ability, but it can hit targets with greater ease.",
```

Comment:
Dispersion is still copied from .22 CB, which is 20 percent worse than standard .22 LR. Would require a reduction in dispersion to be consistent with how shotshells are modeled in-game. It would be reasonable to not make the reduction in dispersion modest relative to .22 LR, given it is most likely to be fired from a rifled barrel.

Edit, and my apologies. I hate it when I hit review by mistake, instead of comment.